common/design_utils.cc

```diff
@@ -71,7 +71,9 @@ void print_utilisation(const Context *ctx)
     }
     std::map<IdString, int> available_types;
     for (auto bel : ctx->getBels()) {
-        available_types[ctx->getBelType(bel)]++;
+        if(!ctx->getBelHidden(bel)) {
```

Comment:
This will need to be added to the Arch API (but I think it's a good thing to add.)

Reply:
This was done in #581